### DISCUSSION TITLE
paddlepaddle 2.3.2 cpu版本下，v4检测模型预测异常

### DISCUSSION THREAD

**Discussion by yangy996** [PaddlePaddle/PaddleOCR#12476] [Q&A]
paddlepaddle 2.3.2 gpu和paddlepaddle 2.5.2 版本
预测结果正常：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/15188456/9f374540-932b-45e8-b271-1a4ca8c07f83)

切换到 paddlepaddle 2.3.2 cpu版本
预测结果异常：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/15188456/8cdab4ea-c01b-4298-a557-33aa75228b3e)

**Comment by SWHL** [comment]
如果问题仍然存在，请拉取最新main分支代码再此尝试
